GetStream/stream-chat-android
# [CAS-1208] Add style attributes to control visibility of attachment gallery menu options

https://stream-io.atlassian.net/browse/CAS-1208

### 🎯 Goal

Users should be able to customize which options they want to display and which ones to hide in the overflow menu options view.

### 🛠 Implementation details

Style attributes have been added that can be used to enable/disable menu options.

### 🧪 Testing

1. Create a custom theme in your app and override the `AttachmentGalleryActivity` theme as mentioned in instructions [here](https://getstream.io/chat/docs/sdk/android/ui/theming/#themes-for-activities).
2. Create a custom style set having these four attributes 
```xml
<style name="CustomAttachmentOptions" parent="StreamUi.AttachmentGallery.Options">
        <item name="streamUiSaveImageEnabled">false</item>
    </style>
```
3. In your custom theme, override the `streamUiAttachmentGalleryOptionsStyle`
`<item name="streamUiAttachmentGalleryOptionsStyle">@style/CustomAttachmentOptions</item>`
4. Save image option should not be enabled and visible when viewing any image attachment.

### ☑️ Checklist


### 🎉 GIF

_Please provide a suitable gif that describes your work on this pull request_
